firestore/angular-rewrite/README.md

```diff
@@ -36,14 +37,72 @@ Follow these steps to setup and run the quickstart:
 
  1. Run and serve the project locally:
     ```bash
     npm start
     ```
  1. Open [http://127.0.0.1:4200](http://127.0.0.1:4200) in your browser and try out the app!
+
+## Configure and run the app with the Firebase Console:
+
+Follow these steps to have a working version of the quickstart running with the Firebase console:
+
+   > **Note:** While **Authentication**, **Storage**, and **Firestore** are offered as free services through the Firebase console, using Firebase **Functions** on the Firebase Console requires a billing plan. The following instructions, therefore, exclude those steps needed to deploy and host functions on the Firebase Console. All other application functionality is conserved.
+
+ 1. Create a new Firebase project in the [Firebase Console](https://console.firebase.google.com)
+
+ 2. On the project homepage, select `Web` under `Get Started By Adding Firebase To Your App` and follow prompts to create your app, making sure to **copy the `firebaseConfig` object** when presented.
  
+ ![Photo of Firebase Project Homepage](images/ProjectHomepage.png)
+
+ ![Photo of firebaseConfig object](images/CopyWebConfig.png)
+
+ 3. With the `firebaseConfig` copied, navigate to `src/environments/environments.ts` and paste the contents of the object into the `prod` field:
+
+ ![Photo of `environments.ts` file with mock firebase web config info copied](images/environments-ts.png)
+
+ 4. On line 79 of `src/app/app.module.ts`, change your environment from `environments.local` to `environments.prod` in the `provideFirebaseApp` function call
+
+ ![Photo of `environments.prod` selection in `src/app/app.module.ts`](images/ProdAppModule.png)
+
+ 5. Back in the Firebase console, enable email and password authentication on your project by doing: **Authentication > SIGN-IN METHOD > Email > Email/Password > SAVE**
+
+ ![Photo of the enable email/password screen on Firebase Authentication page](images/Enable-Email.png)
+
+ 6. In the Firebase console, enable Firestore on your project by clicking **Create Database** in the **Cloud Firestore** section of the console and answering all prompts.
+
+   * When prompted, select "Start in Test Mode"
+
+ ![Photo of Firebase Firestore Setup](images/EnableFirestore.png)
+
+ 7. Simmilarly, enable Cloud Storage on your project by clicking **Get Started** in the **Storage** section of the console and answering all prompts.
+
+   * When prompted, select "Start in Test Mode"
+
+ ![Photo of Firebase Storage Setup](images/EnableStorage.png)
+
+ 8. Set the CLI to use the project you created in Step 1:
+
+ ```bash
+ firebase --use add
+ ```
+
+ 9. Deploy the firebase security rules and indexes
+
+ ```bash
+ firebase deploy --only firestore
+ ```
+
+ > **Note:** If you've opted to set up a billing plan and wish to deploy functions as well, replace this command with simply `firebase deploy`.
```

Comment:
Note the missing functionality, also let them know that this isn't going to cost them anything, point to the free tier doc so they know they probably won't be charged: https://firebase.google.com/pricing